README.md

```diff
@@ -193,7 +193,7 @@ One easy is to check in the CoW Explorer. You can create a CoW Explorer link if
 
 ```js
 // View order in explorer
-console.log(`https://explorer.cow.fi/rinkeby/orders/${orderId}`)
+console.log(`https://explorer.cow.fi/gc/orders/${orderId}`)
```

Comment:
There are two more places where chainId 4 is used on the README

1. https://github.com/cowprotocol/cow-sdk/pull/76/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L91
2. https://github.com/cowprotocol/cow-sdk/pull/76/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R301

(you have to expand the file to see it)